BLSQ/iaso
# IA-3756 Fix `delete_accounts.py`

IA-3756 Fix `delete_accounts.py`.

Related JIRA tickets : [IA-3756](https://bluesquare.atlassian.net/browse/IA-3756)

## Changes

Fix `delete_accounts.py`:

- delete `TenantUser` related to projects
- delete `OrgUnitChangeRequestConfiguration` related to projects

Use `on_delete=models.CASCADE`:

- on `TenantUser.main_user` and `TenantUser.account_user`
- on `OrgUnitChangeRequestConfiguration.project

## How to test

Unfortunately there are no unit tests for this code.

Only @mbayangdiop95 will be able to confirm that this patches fix the script.

[IA-3756]: https://bluesquare.atlassian.net/browse/IA-3756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
